SubmissionExample.ipynb

```diff
@@ -0,0 +1,592 @@
+   "source": [
+    "!cd v0.5/classification_and_detection; python setup.py develop"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Set Working Directory"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "/mnt/sdb/home/ipython/notebooks/inference/v0.5/classification_and_detection\n"
+     ]
+    }
+   ],
+   "source": [
+    "%cd v0.5/classification_and_detection"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Download data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%bash\n",
+    "\n",
+    "## download imagenet and coco.\n",
+    "## we keep a copy under /data so just symlink it.\n",
+    "mkdir data\n",
+    "ln -s  /data/imagenet2012 data/\n",
+    "ln -s  /data/coco data/"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Download models"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%bash\n",
+    "\n",
+    "mkdir models\n",
+    "\n",
+    "# resnet50\n",
+    "wget -q https://zenodo.org/record/2535873/files/resnet50_v1.pb -O models/resnet50_v1.pb \n",
+    "wget -q https://zenodo.org/record/2592612/files/resnet50_v1.onnx -O models/resnet50_v1.onnx\n",
+    "\n",
+    "# mobilenet\n",
+    "wget -q https://zenodo.org/record/2269307/files/mobilenet_v1_1.0_224.tgz -O models/mobilenet_v1_1.0_224.tgz\n",
+    "cd models; tar zxvf mobilenet_v1_1.0_224.tgz\n",
+    "wget -q https://zenodo.org/record/3157894/files/mobilenet_v1_1.0_224.onnx -O models/mobilenet_v1_1.0_224.onnx\n",
+    "\n",
+    "# ssd-mobilenet\n",
+    "wget -q http://download.tensorflow.org/models/object_detection/ssd_mobilenet_v1_coco_2018_01_28.tar.gz -O models/ssd_mobilenet_v1_coco_2018_01_28.tar.gz\n",
+    "cd models; tar zxvf ssd_mobilenet_v1_coco_2018_01_28.tar.gz; mv ssd_mobilenet_v1_coco_2018_01_28/frozen_inference_graph.pb ssd_mobilenet_v1_coco_2018_01_28.pb\n",
+    "wget -q https://zenodo.org/record/3163026/files/ssd_mobilenet_v1_coco_2018_01_28.onnx -O models/ssd_mobilenet_v1_coco_2018_01_28.onnx \n",
+    "\n",
+    "# ssd-resnet34\n",
+    "wget -q https://zenodo.org/record/3345892/files/tf_ssd_resnet34_22.1.zip -O models/tf_ssd_resnet34_22.1.zip\n",
+    "cd models; unzip tf_ssd_resnet34_22.1.zip; mv tf_ssd_resnet34_22.1/resnet34_tf.22.1.pb .\n",
+    "wget -q https://zenodo.org/record/3228411/files/resnet34-ssd1200.onnx -O models/resnet34-ssd1200.onnx"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Run benchmarks using the reference implementation\n",
+    "\n",
+    "Lets pretent we want to submit mobilenet on a desktop machine with a Gtx-1080 gpu using tensorflow. \n",
```

Comment:
nit: "pretend"
nit: "GTX-1080" or "gtx-1080"

Reply:
Pretend? We are going to do it for real :)

Reply:
changed the text to ```Lets prepare a submission```